website/data/plugins-manifest.json

```
@@ -309,5 +309,12 @@
     "repo": "hashicorp/packer-plugin-yandex",
     "version": "latest",
     "pluginTier": "community"
+  },
+  {
+    "title": "UpCloud",
+    "path": "upcloud",
+    "repo": "UpCloudLtd/packer-plugin-upcloud",
+    "version": "latest",
+    "pluginTier": "community"
```

By default the "Edit this page" in the footer for documentation page refers to the main branch. Since your repo uses master we need to also set the following attribute.
```suggestion
    "pluginTier": "community",
    "sourceBranch": "master"
```

---

@nywilken Thank you for feedback. I'll push changes to fix this.